src/gui/MainWindow.cpp

```diff
@@ -112,7 +112,7 @@ MainWindow::MainWindow() :
 	sideBar->appendTab( new FileBrowser(
 				confMgr->userProjectsDir() + "*" +
 				confMgr->factoryProjectsDir(),
-					"*.mmp *.mmpz *.xml *.mid",
+					"*.mmp *.mmpz *.xml *.mid *.mpt",
```

Comment:
Out of scope, but we should add these extensions dynamically based on the available import filter plugins.

Reply:
My templates directory was showing up empty without this change, and since it is a valid project file I figured it should show up under Projects, but if you really think it's out of scope I will remove it.

Reply:
Sorry, I didn't mean the change as you have it is out of scope, but rather my suggestion. Definitely keep this for now.